src/Sulu/Component/Util/Tests/Unit/SortUtilsTest.php

```diff
@@ -174,8 +172,6 @@ public function testSortLocaleAwareSimpleArray(): void
         $result = SortUtils::sortLocaleAware($array, 'de');
 
         if (!\class_exists(\Collator::class)) {
-            $this->addWarning('Could not test sorting with \\Collator, because intl extension is not loaded');
```

Comment:
???

Reply:
This method does not exist anymore in Phpunit 10 (as the matching for warning has been removed)

Reply:
Should we do a `$this->markTestAsSkipped` or what would you recommend. Whatever we do we just adopt it in the 2.5 branch the same way.